README.md

```diff
@@ -69,7 +69,27 @@ dependencies {
 
 ### Usage
 
-// TODO
+Register `Search Manager`
+```java
+SearchManager searchManager = new SearchManager();
+```
+
+Register sources
+```java
+var spotify = new SpotifySourceManager(clientId, clientSecret, spDc, countryCode, () -> audioPlayerManager, DefaultMirroringAudioTrackResolver)
+var youtube = new YoutubeSearchManager(() -> audioPlayerManager, region);
+
+searchManager.registerSearchManager(spotify);
+searchManager.registerSearchManager(youtube);
```

Comment:
please don't include lavasrc examples in here, I don't want to maintain them at 2 places